### ISSUE TITLE
@PrivacyBadger and @HTTPSEverywhere extensions on chrome

### ISSUE THREAD

**Issue by DianaMPG** [openspending/openspending#1236]
@PrivacyBadger and @HTTPSEverywhere extensions on chrome when uploading file to @openspending won't work.

**Comment by Hazwany** [comment]
Hi @DianaMPG, thanks for flagging this issue. [Earlier today](https://twitter.com/nanyjharon/status/864292260078211072), I was facing an error (as shown below) when I was trying to upload a file to OpenSpending. While I was uploading the file, the Privacy Badger and HTTPS Everywhere chrome extensions were enabled.

![screen shot 2017-05-16 at 9 30 45 am](https://cloud.githubusercontent.com/assets/11497391/26100947/f89222be-3a61-11e7-8f1b-ba2dcc4d2665.png)

Then, I tried uploading again to OS but disabled the 2 extensions, it worked fine without any error. And I tried the same process for 3rd time but enabled 2 extensions, it worked fine with just one error.

![screen shot 2017-05-16 at 6 33 39 pm](https://cloud.githubusercontent.com/assets/11497391/26102119/3e901f1a-3a66-11e7-8c0a-800b1c5ad11a.png)

I couldn't figured out what went wrong in the first place but it works fine now (?)
